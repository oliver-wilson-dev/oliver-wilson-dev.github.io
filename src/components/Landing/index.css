.landing {
  align-items: center;
  display: flex;
  flex-direction: column;
  margin: 10px 30px;
}

.landingHeader {
  font-size: 5em;
  margin: 0;
  margin-top: 10px;
  background-clip: text;
  -webkit-background-clip: text;
  letter-spacing: 2px;
  text-align: center;
  background: #f8f8f8;
  width: 100%;
}

.landingSubHeader {
  font-size: 32px;
  margin: 15px 0;
  text-align: center;
  font-weight: 300;
  letter-spacing: 4px;
  text-transform: lowercase;
}

@media all and (min-width: 786px) {
  .landing {
    padding-top: 100px;
  }
}

@media (min-width: 786px) {
  .landing--header {
  max-width: 800px;
  }
}


@media (min-width: 0px) and (max-width: 767px) {
  .landing--header {
    font-size: 3em;
  }

  .landing--sub-header {
    font-size: 28px;
  }
}