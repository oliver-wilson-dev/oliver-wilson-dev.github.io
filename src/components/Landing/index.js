import React from 'react';
import cn from 'classnames';
import styles from './index.css';

export default () => (
  <div className={cn(styles.container, styles.landing)}>
    <h1 className={styles.landingHeader}>Oliver Wilson</h1>
    <h2 className={styles.landingSubHeader}>
      Web developer
    </h2>
  </div>
);
