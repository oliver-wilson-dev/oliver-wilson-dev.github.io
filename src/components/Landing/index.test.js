import React from 'react';
import { shallow } from 'enzyme';
import Landing from '.';

const render = () => shallow(<Landing />);

describe('<Landing/>', () => {
  it('should exist', () => {
    expect(render().exists()).toBe(true);
  });

  it('should render correctly', () => {
    expect(render()).toMatchSnapshot();
  });
});
