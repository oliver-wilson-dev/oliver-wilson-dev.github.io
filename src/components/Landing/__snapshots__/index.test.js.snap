// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing/> should render correctly 1`] = `
<div
  className="container landing"
>
  <h1
    className="landingHeader"
  >
    Oliver Wilson
  </h1>
  <h2
    className="landingSubHeader"
  >
    Web developer
  </h2>
</div>
`;
